src/charm.py

```diff
@@ -84,6 +87,11 @@ def __init__(self, *args: Any) -> None:
         self._db_name = f"{self.model.name}_{self.app.name}"
         self._db_relation_name = "pg-database"
         self._login_ui_relation_name = "ui-endpoint-info"
+        self._prometheus_scrape_relation_name = "metrics-endpoint"
+        self._loki_push_api_relation_name = "logging"
+        self._hydra_service_command = "hydra serve all"
```

Comment:
why do you put the serve command in a variable?

Reply:
I try to keep the code in our charms uniform, and store the specifics in variables.